src/main/java/com/piekoszek/nowaksharedrent/config/UndefinedExceptionHandlerController.java

```diff
@@ -1,17 +1,12 @@
 package com.piekoszek.nowaksharedrent.config;
 
-import com.piekoszek.nowaksharedrent.response.MessageResponse;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ControllerAdvice;
-import org.springframework.web.bind.annotation.ExceptionHandler;
-import org.springframework.web.context.request.WebRequest;
 
 @ControllerAdvice
 class UndefinedExceptionHandlerController {
 
-    @ExceptionHandler(Throwable.class)
-    public ResponseEntity<Object> handleThrowableException (Throwable ex, WebRequest request) {
-        return new ResponseEntity<>(new MessageResponse("Undefined error! Sorry, we are working on that"), HttpStatus.BAD_REQUEST);
-    }
+//    @ExceptionHandler(Throwable.class)
```

Comment:
Why is it commented?

Reply:
I need some mechanism to log every undefined error. The problem is that I have no idea what's going on during debugging the app and I am commenting it everytime I'd like to check exception message. This time I forgot to uncomment it afterwards.

Reply:
You can log.error() this exception and you should see stack trace.

Reply:
Done